### ISSUE TITLE
Multiple criteria descriptions are missing "show full description"

### ISSUE THREAD

**Issue by rickdoesburg** [w3c/wai-wcag-em-report-tool#171]
**Describe the bug**
Multiple WCAG Criteria in the audit are missing the 'show full description' button to trigger more context during the reporting.

For example:
2.5.8
3.2.6
3.3.4
3.3.7

**To Reproduce**
1. Go to the WCAG-EM Reporting tool
2. Start an audit, go to step 4.
3. Scroll down to the sections mentioned above
4. See missing context

**Expected behavior**
I expect there to be a button which expands more context just like the other criteria have.

**Screenshots**
![image](https://github.com/user-attachments/assets/1f98c3e6-0e92-4fbe-8473-6ffbf8ffc826)
![image](https://github.com/user-attachments/assets/5ff47c1d-5272-4327-9f80-cad695a9099e)

**Desktop (please complete the following information):**
 - OS: Mac OS 14.6.1
 - Browser: Firefox
 - Version: 131
